rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1378,57 +1378,80 @@ void spu_thread::do_dma_transfer(const spu_mfc_cmd& args)
 					continue;
 				}
 
+				if (raddr == (eal & -128) && time0 != rtime)
+				{
+					// Validate rtime for read data
+					set_events(SPU_EVENT_LR);
+					raddr = 0;
+				}
+
+				alignas(64) u8 temp[128];
```

Comment:
Temporary storage, similar to a recent bugfix in GETLLAR which doesnt rely on LS to store constant data because its volatile.